assets/sidecars/volumeattachment_provisioner_role.yaml

```diff
@@ -0,0 +1,8 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: volumeattachment-external-provisioner-role
+rules:
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["volumeattachments"]
+    verbs: ["get", "list", "watch"]
```

Comment:
We have to distinguish them because "verbs" lists are different: `["get", "list", "watch"]` for provisioner, and `["get", "list", "watch", "update", "patch"]` for attacher. It would impact security if we let provisioner to modify volumeattachments although it needs only read-only access to them.

Reply:
The provisioner and attacher use the same service account, so there is no impact on security. IMO having the permissions that detailed will impact readability.

Reply:
@jsafrane , how can we apply your explanation about provisioner and attacher to the manila usecase? We don't have `attacher_role.yaml` there, but in `provisioner_role.yaml`:

```
  - apiGroups: ["storage.k8s.io"]
    resources: ["volumeattachments"]
    verbs: ["get", "list", "watch"]
```
(https://github.com/openshift/csi-driver-manila-operator/blob/master/assets/rbac/provisioner_role.yaml#L24-L26)

Reply:
It looks like a bug in Manila then.

Reply:
or, maybe not...  Keep the reader role there, we'll see when we get to individual drivers.